lte/gateway/deploy/agw_install.sh

```diff
@@ -70,24 +77,64 @@ if [ "$KVERS" != "4.9.0-9-amd64" ]; then
   NEED_REBOOT=1
 fi
 
+# configure environment variable defaults needed for ansible
+ANSIBLE_VARS="PACKAGE_LOCATION=/tmp"
+if [ -n "${REPO_HOST}" ]; then
+    if [ -z "${REPO_PROTO}" ]; then
+        REPO_PROTO=http
+    fi
+    if [ -z "${REPO_DIST}" ]; then
+        REPO_DIST=stretch-stable
+    fi
+    if [ -z "${REPO_COMPONENT}" ]; then
+        REPO_COMPONENT=main
```

Comment:
I don't see the point to split the repo location + proto + dist + component in 3 vars

Reply:
I needed to do this when debugging bare metal agw provisioning, will likely be needed again for similar reasons (custom deploy test, change repo layout, ...)